flixel/input/gamepad/FlxGamepadAnalogStick.hx

```diff
@@ -42,12 +42,12 @@ class FlxGamepadAnalogStick
 		if (settings == null)
 			return;
 
-		mode = (settings.mode != null) ? settings.mode : ONLY_ANALOG;
+		mode = (settings.mode != null) ? settings.mode : BOTH;
 		rawUp = (settings.up != null) ? settings.up : -1;
 		rawDown = (settings.down != null) ? settings.down : -1;
 		rawLeft = (settings.left != null) ? settings.left : -1;
 		rawRight = (settings.right != null) ? settings.right : -1;
-		digitalThreshold = (settings.threshold != null) ? settings.threshold : -1;
+		digitalThreshold = (settings.threshold != null) ? settings.threshold : 0.5;
```

Comment:
To clarify in this situation: since the default setting everywhere was to only report Analog, I'm pretty sure nobody was actually accessing this behavior. Accessing analog-to-digital virtual stick directions was buried in the API, essentially undocumented, and only worked on legacy. This PR is basically the first time the feature's actually been exposed to daylight in any practical sense.